src/lib/ext-activitylog.js

```diff
@@ -0,0 +1,143 @@
+import { save, load } from '../lib/save-load.js';
+
+export default class ActivityLog {
+  constructor() {
+    this.logs = [];
+    this.loadedLogs = [];
+    this.rowIndex = 0;
+
+    this.loadLogBtn = document.getElementById('loadLogBtn');
+    this.logInputFile = document.getElementById('logInputFile');
+    this.saveLogBtn = document.getElementById('saveLogBtn');
+    this.showLogDetailWrapper = document.querySelector(
+      '.show-log-detail-wrapper'
+    );
+    this.showLogDetails = document.getElementById('showLogDetails');
+    this.showLoadedLog = document.getElementById('showLoadedLog');
+    this.logTableWrapper = document.querySelector('.log-table');
+    this.table = document.querySelector('table');
+    this.notice = document.querySelector('.notice');
+    this.closeBtn = document.querySelector('.close');
```

Comment:
That's a reasonable point, and it may as well be said for ext-popup.js.

In the last meeting we briefly discussed of using the WebComponents API for the classes that are going to manage the DOM elements, but we agreed to defer that to follow ups. 
As part of those follow ups we may also move the ESM modules that are going to export WebComponents classes into a separate lib/web-components/ directory (and tweak the class names a bit, e.g. by adding a `*Element' suffix).